runtime/ftplugin/man.vim

```diff
@@ -20,13 +20,13 @@ setlocal wrap breakindent linebreak
 setlocal nonumber norelativenumber
 setlocal foldcolumn=0 colorcolumn=0 nolist nofoldenable
 
+setlocal tagfunc=man#goto_tag
+
 if !exists('g:no_plugin_maps') && !exists('g:no_man_maps')
   nnoremap <silent> <buffer> j          gj
   nnoremap <silent> <buffer> k          gk
   nnoremap <silent> <buffer> gO         :call man#show_toc()<CR>
-  nnoremap <silent> <buffer> <C-]>      :Man<CR>
   nnoremap <silent> <buffer> K          :Man<CR>
-  nnoremap <silent> <buffer> <C-T>      :call man#pop_tag()<CR>
```

Comment:
I suppose `K` should be mapped to `<C-]>` now? 

But I'll leave it as-is, since I noticed a difference: with e.g. `:Man git-notes`, cursor at the caret position:

```
GIT
       Part of the git(1) suite
                        ^
```

`K` (mapped to `:Man<CR>`) follows the `git(1)` reference correctly ,whereas `<c-]>` searches for `suite`.

Reply:
Yeah that's an interesting one. I think `iskeyword` including `'('` and `')'` when `filetype=man` is the right fix to make for this. I've opened a #11457 to sort this.

I think having `keywordprg=:Man` is okay though, it feels more natural than adding a mapping and ignoring any potential custom `'keywordprg'` a user might have set for man pages. Would you agree?

Reply:
> having keywordprg=:Man is okay though, it feels more natural than adding a mapping

Oh, yes. So the K mapping was  an oversight , or at least it is after 'tagfunc' was implemented.